packages/console/src/components/ApplicationIcon/index.tsx

```diff
@@ -1,17 +1,18 @@
 import type { ApplicationType } from '@logto/schemas';
-import { AppearanceMode } from '@logto/schemas';
+import { useContext } from 'react';
 
 import { darkModeApplicationIconMap, lightModeApplicationIconMap } from '@/consts';
-import { useTheme } from '@/hooks/use-theme';
+import { AppThemeContext } from '@/contexts/AppThemeProvider';
+import { Theme } from '@/types/theme';
 
 type Props = {
   type: ApplicationType;
   className?: string;
 };
 
 const ApplicationIcon = ({ type, className }: Props) => {
-  const theme = useTheme();
-  const isLightMode = theme === AppearanceMode.LightMode;
+  const { theme } = useContext(AppThemeContext);
+  const isLightMode = theme === Theme.LightMode;
```

Comment:
Just curious since you mentioned on the PR body, Theme is not equivalent to the ApperanceMode. What is the difference?

Reply:
The `ApperanceMode` is used for user settings with three values: `LigthMode`, `DarkMode`, and `SyncWithSystem`.